source/common/src/tglobal.c

```diff
@@ -511,6 +512,7 @@ static int32_t taosAddServerCfg(SConfig *pCfg) {
   if (cfgAddInt32(pCfg, "mqRebalanceInterval", tsMqRebalanceInterval, 1, 10000, 1) != 0) return -1;
   if (cfgAddInt32(pCfg, "ttlUnit", tsTtlUnit, 1, 86400 * 365, 1) != 0) return -1;
   if (cfgAddInt32(pCfg, "ttlPushInterval", tsTtlPushInterval, 1, 100000, 1) != 0) return -1;
+  if (cfgAddBool(pCfg, "enableTtlByCtime", tsEnableTtlByCtime, 0) != 0) return -1;
```

Comment:
Please make the configuration  `enableTtlByCtime` more understandable.

Reply:
Renamed `enableTtlByCtime`  -> `ttlChangeOnWrite` by suggestion